### ISSUE TITLE
TypeError:Argument 3 passed to Keboola\Google\ClientBundle\Google\RestApi::decideRetry() must implement interface Psr\Http\Message\ResponseInterface

### ISSUE THREAD

**Issue by pivnicek** [keboola/google-client-bundle#4]
https://connection.keboola.com/admin/utils/logs?file=2019/10/01/02/2019-10-01-02-33-27-5d929ed77d5b3-exception

**Comment by MiroCillik** [comment]
I think this is already solved in the latest version, but the components are not updated to it:
https://github.com/keboola/google-client-bundle/blob/master/src/Google/RestApi.php#L309.

Well, the Google Sheets Client finally is, but now I have to update Google Sheets and Drive Writers.

**Comment by pivnicek** [comment]
yes, sorry I noticed that when I checked the history here.  But I guess we can use this issue to checklist the updating of the components.
